src/SearchResults/index.js

```diff
@@ -286,7 +285,11 @@ function SearchResults(state, results) {
    * sum of the processing time of all the queries
    * @member {number}
    */
-  this.processingTimeMS = sumBy(results, 'processingTimeMS');
+  this.processingTimeMS = results.reduce(function(sum, result) {
+    return result.processingTimeMS === undefined
```

Comment:
For which reason the `processingTimeMS` will not be there? I think it's always there (see [doc](https://www.algolia.com/doc/api-reference/api-methods/search/#method-response-processingtimems)).

Reply:
@tkrugg pointed out you can filter it out: https://www.algolia.com/doc/api-reference/api-parameters/responseFields/?language=js#usage-notes